diff --git a/src/deps/CMakeLists.txt b/src/deps/CMakeLists.txt
index 18c7c40..c287fe2 100644
--- a/src/deps/CMakeLists.txt
+++ b/src/deps/CMakeLists.txt
@@ -1,4 +1,3 @@
-add_subdirectory(pybind11)
 if (NOT OTIO_CXX_NOINSTALL)
     install(FILES any/any.hpp DESTINATION include/opentimelineio/deps/any)
     install(FILES optional-lite/include/nonstd/optional.hpp
@@ -6,7 +5,7 @@ if (NOT OTIO_CXX_NOINSTALL)
 endif (NOT OTIO_CXX_NOINSTALL)
 
 # detect if the submodules haven't been updated
-set(DEPS_SUBMODULES any optional-lite pybind11 rapidjson)
+set(DEPS_SUBMODULES any optional-lite rapidjson)
 foreach (submodule IN LISTS DEPS_SUBMODULES)
     file(GLOB SUBMOD_CONTENTS ${submodule})
     list(LENGTH SUBMOD_CONTENTS SUBMOD_CONTENT_LEN)
